easybuild/easyconfigs/g/GCC/GCC-5.4.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'Bundle'
+
+name = 'GCC'
+version = '5.4.0'
+
+binutilsver = '2.26'
+#versionsuffix = '-%s' % binutilsver
```

Comment:
@valtandor Why are you excluding the binutils version from the `versionsuffix` here? It really helps in understanding what this provides imho...